libs/challenge-registry/challenge-search/src/lib/challenge-search-query.ts

```diff
@@ -5,12 +5,13 @@ import {
   ChallengeIncentiveType,
   ChallengeSubmissionType,
   DateRange,
+  Challenge,
 } from '@sagebionetworks/challenge-registry/api-client-angular-deprecated';
 
 export interface ChallengeSearchQuery {
   limit?: number;
   offset?: number;
-  // sort?: 'createdAt' | 'updatedAt';
+  sort?: keyof Challenge;
```

Comment:
Does that mean that the sort value need to match a property of the challenge object?

Reply:
that's correct.